spec/e2e/accessibility_spec.js

```diff
@@ -1,18 +1,16 @@
+import { describe, test, it } from 'node:test';
+import assert from 'node:assert/strict';
 import { AxePuppeteer } from '@axe-core/puppeteer';
-import { toHaveNoViolations } from 'jest-axe';
-import { page as globalPage, goto } from './support/browser';
-import { getCandidateLinks, toNotHaveTargetBlank } from './support/target-blank';
-
-expect.extend(toHaveNoViolations);
-expect.extend({ toNotHaveTargetBlank });
+import { getURL } from './support/browser';
+import { getCandidateLinks } from './support/target-blank';
 
 /** @type {RegExp[]} */
 const EXCLUDE_PATTERNS = [
   /\.pdf$/, // Puppeteer Chromium cannot preview PDF files
   /admin/,
 ];
 
-describe('accessibility', () => {
+describe('accessibility', { concurrency: 4 }, () => {
```

Comment:
WDYT of making concurrency overridable with an env var?
```suggestion
describe('accessibility', { concurrency: parseInt(process.env['CONCURRENCY'] || 4) }, () => {
```

Reply:
Looking through [the docs](https://nodejs.org/api/test.html#test-runner-execution-model), it might make more sense to code this as `concurrency: true` and then allow the value to be assigned through `--test-concurrency`. I don't see anything about an environment variable, though that would be nice (easier to override) if it existed.

Reply:
I think something's wrong with how the concurrency is working, since it's [documented](https://nodejs.org/api/cli.html#--test-concurrency) to only run as `os.availableParallelism() - 1`. On my machine, that's `7`, but it looks like it's trying to run all pages at the same time when passing `concurrency: true`. I'll poke at this a bit.

Reply:
I think the documentation above is referring specifically to files run in parallel, though I also don't see files run in parallel. As far as concurrency within a test, the [documentation for the `test` option](https://nodejs.org/api/test.html#testname-options-fn) could reinforce what I'm seeing with trying to run them all at the same time:

>If `true`, all scheduled asynchronous tests run concurrently within the thread. 

In any case, after spending a good chunk of my afternoon banging my head against this, I went with a solution using `process.env` similar to what you suggested in 6b33dbdd3f5a0c7e95e56bf2f5e556001bdbbac5.